python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -261,15 +343,19 @@ def __init__(
         username: str,
         password: str,
         environment: NCMECEnvironment,
+        member_id: t.Optional[str] = None,
+        reasons_map: t.Dict[str, t.List[t.Dict[str, str]]] = {},
```

Comment:
blocking: We won't know this at client construction time, and can poll it afterwards.